guides/low_level_api.md

```diff
@@ -0,0 +1,60 @@
+# Low-level API
```

Comment:
We have a lot of confusion around what's "high" or "low" level. I would keep the terminology to something more specific like in https://github.com/opensearch-project/opensearch-py/blob/main/guides/json.md. This API allows you to make (raw) HTTP requests, so I would call it as such.

I would rename the file to json.md to be consistent with other clients, too.